addons/web/static/tests/views/list/list_view.test.js

```diff
@@ -19066,3 +19066,16 @@ test(`multi_edit: edit field with operator with localization`, async () => {
     expect(`table tr:eq(3) td[name=${field}]`).toHaveText("34");
     expect(`table tr:eq(4) td[name=${field}]`).toHaveText("-8");
 });
+
+test.tags("desktop");
+test(`list_view: leave edition when click on searchbar`, async () => {
+    await mountView({
+        resModel: "foo",
+        type: "list",
+        arch: `<list editable="top"><field name="foo"/></list>`,
+    });
+    await contains(".o_data_cell").click();
+    await contains(".o_data_cell input").edit("blabla", { confirm: false });
+    await contains(".o_searchview").click();
```

Comment:
does the test fail without your diff? As far as I can see, it already works when clicking in the search view. I only have the issue when clicking on the toggler (`.o_searchview_dropdown_toggler`)